modules/sdk/project-templates/project-templates-spring-mvc-portlet/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/spring-context/portlet-application-context.xml

```diff
@@ -31,15 +31,19 @@
 		<property name="suffix" value=".html"/>
 		<property name="templateMode" value="HTML"/>
 	</bean>
-#if (${framework.equals("portletmvc4spring")})
+#if (${jakartaCompatible.equals("true")})
+	<bean class="org.thymeleaf.spring6.SpringTemplateEngine" id="templateEngine">
+#elseif (${framework.equals("portletmvc4spring")})
```

Comment:
let's set a `springPackage` var above for this as well instead of splitting the whole line, so the result will look like: 
```
<bean class="org.thymeleaf.${springPackage}.SpringTemplateEngine" id="templateEngine">
```